src/Kafka/ManageKafkaPermissions/components/ResourcePrefix.tsx

```diff
@@ -0,0 +1,38 @@
+import { FormGroup, TextInput, ValidatedOptions } from "@patternfly/react-core";
+import { useTranslation } from "react-i18next";
+
+export type ResourcePrefixProps = {
+  value: string | undefined;
+  onChangeValue: (value: string | undefined) => void;
+  invalid: boolean;
+};
+
+export const ResourcePrefix: React.VFC<ResourcePrefixProps> = ({
+  value,
+  onChangeValue,
+  invalid,
+}) => {
+  const { t } = useTranslation(["manage-kafka-permissions"]);
+
+  const onPrefixValueChange = (value: string | undefined) => {
+    onChangeValue(value);
+  };
+
+  return (
+    <FormGroup
+      validated={invalid ? ValidatedOptions.error : ValidatedOptions.default}
+      helperTextInvalid={t("common:required")}
+      fieldId={"resource-prefix-input"}
+    >
+      <TextInput
+        placeholder={t("resourcePrefix.placeholder_text")}
```

Comment:
We use a very similar component if the user chooses "Is" instead of "Starts with". Should this component support both options? 
<img width="404" alt="image" src="https://user-images.githubusercontent.com/21063328/158890615-7aacabdf-6c5b-4f00-b117-0c20afa81f8c.png">

Something to note is that the validation messages we display could vary depending on "is" vs "starts with". For example, ".." is a valid prefix, but not a valid name. 

So back to @riccardo-forina's previous question - maybe there should be the following components: 
* a typeahead select component without the form group
* a prefix typeahead inside a form group with all the validation messages
* a name typeahead inside a form group with all the validation messages